icu/meta.yaml

```diff
@@ -9,6 +9,12 @@ source:
   fn: icu4c-54_1-Win32-msvc10.zip # [win32]
   url: http://download.icu-project.org/files/icu4c/54.1/icu4c-54_1-Win32-msvc10.zip # [win32]
   md5: 2419927dae9f0ac2498b10de657ff6d3 # [win32]
+  fn: icu4c-54_1-RHEL6-x64.tgz # [linux64]
```

Comment:
We try to build everything in CentOS 5. Are there sources available for that platform?